script.js

```diff
@@ -0,0 +1,7 @@
+var showButton = document.getElementsByClassName('button');
+
+for (var i = 0; i <= showButton.length; i++) {
+		
+	 	showButton.innerText = 'The text inside buttons 1 2 3';
```

Comment:
it is not true, showButton is an array - innerText isn't a property of array. you need to take from array every button and use innerText on every single button ;) for example: `showButton[0]` is the first button in your array :)